App.rb

```diff
@@ -5,11 +5,11 @@
 module Shop
 
   PRODUCTS = [
-    Product.new("Coffe",20),
-    Product.new("Tea",30),
-    Product.new("Chocolate",12),
-    Product.new("Cookies",50),
-    Product.new("Fireworks",300)
+    Product.new("Pikachu",8200, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png"),
+    Product.new("Bulbasaur",3000, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png"),
+    Product.new("Squirle",3900, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/007.png"),
+    Product.new("Jigglypuff",2400, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/039.png"),
+    Product.new("Snorlax",5800, "http://assets.pokemon.com/assets/cms2/img/pokedex/full/143.png")
```

Comment:
@smt116 I couldn't add pictures from my project folder, so I choose url. But I see that using an url from other sites is a bad practise; I'll use image hosting next time.